sources/packages/backend/apps/db-migrations/src/sql/SequenceControl/Rename-sequence-name.sql

```diff
@@ -0,0 +1,3 @@
+UPDATE sims.sequence_controls
+SET sequence_name = 'Part Time_DISBURSEMENT_DOCUMENT_NUMBER'::varchar(100)
```

Comment:
Please remove the explicit cast `varchar(100)` or clarify if there is a strong reason to keep it.